wkcuber/wkcuber/convert_zarr.py

```diff
@@ -117,9 +117,6 @@ def _zarr_chunk_converter(
 
     slices = bounding_box.to_slices()
     zarr_file = zarr.open(source_zarr_path, "r")
-    shape = zarr_file.shape
-
-    slices = tuple(slice(s.start, min([s.stop, s_])) for s, s_ in zip(slices, shape))
```

Comment:
This was superfluous, since the chunked bounding box was already set to the zarr-files' shape, see https://github.com/scalableminds/webknossos-libs/blob/1192ed5d757c53804e988c2681c8fb93d491cdf9/wkcuber/wkcuber/convert_zarr.py#L158